File: import-json.cs
using System;
using System.Collections.Generic;
using System.IO;
using Newtonsoft.Json;

public class DeserializeJsonFromFile
{
    public static void Main()
    {
        string jsonFilePath = "path/to/your/file.json";
        string json = File.ReadAllText(jsonFilePath);
        var data = JsonConvert.DeserializeObject<Dictionary<string, List<string>>>(json);

        foreach (var item in data)
        {
            Console.WriteLine($"{item.Key}: {string.Join(", ", item.Value)}");
        }
    }
}

File: import-json.rs
use serde_json::Value;
use std::fs;

fn main() {
    let json_file_path = "path/to/your/file.json";
    let json_data = fs::read_to_string(json_file_path).expect("Unable to read file");
    let data: Value = serde_json::from_str(&json_data).expect("Unable to parse JSON");

    println!("{:?}", data);
}
